### PR TITLE
Add reset password feature

### PR DESCRIPTION
**Pull request by han-gyeong**
## Description
This PR is about #611. This PR includes reset password feature.

image
"Reset Password" button has been added.

image
To reset password, user needs to input their id and email.

image
If user found with information, new temporary password will be sent to their email.

image
This is an example of email. It includes User ID, new password, modified date, requested IP.

image
If user can not be found with information, error message appears.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/session/login.jsp [hunk 1]
```diff
@@ -301,11 +362,12 @@
 									<span class="checkSet"><input type="checkbox" id="saveID" /><label for="saveID">SAVE ID</label></span>
 									<strong><a class="btnRegist" id="btnRegist">SignUp</a></strong>
 								</span>
+								<span class="options">
+									<strong><a id="btnResetPwd">Reset Password</a></strong>
+								</span>
```
Comment (jongwooo) [inline src/main/webapp/WEB-INF/views/session/login.jsp:367]: Please move this code into the span tag where class name is "joinGo"(line 361-364).

**Review by jongwooo — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/views/session/login.jsp (hunk 1 above)
Comment (han-gyeong) [reply src/main/webapp/WEB-INF/views/session/login.jsp]: image
Changed, thanks for the review!

**Comment by han-gyeong** [comment]
@soimkim How can i handle this conflict? In CoConstDef.java, all i add is
```java
	public static final String CD_MAIL_TYPE_RESET_USER_PASSWORD = "817";
```
(On line 561)

**Comment by han-gyeong** [comment]
@soimkim This pull request is gonna be closed tomorrow. Can i get any review or comment to be merged?

**Comment by soimkim** [comment]
@han-gyeong , sorry for the late review. I'll review next week.
I'll make sure this issue isn't closed before review.

**Comment by soimkim** [comment]
@Min-Kyungsun , Please review this PR.

**Comment by Min-Kyungsun** [comment]
@han-gyeong 
It is difficult to identify modifications in the below files because of indentation.
src/main/java/oss/fosslight/common/CoConstDef.java
src/main/java/oss/fosslight/controller/UserController.java

Please, push the commit again so that the modifications is identified clearly.

**Comment by Min-Kyungsun** [comment]
@han-gyeong 
Indentation issue in src/main/java/oss/fosslight/common/CoConstDef.java has not fixed yet.

**Comment by han-gyeong** [comment]
@Min-Kyungsun Everything fixed, please review when possible :)

**Pull request merged**